Tests/GRPCTests/ClientTimeoutTests.swift

```diff
@@ -169,7 +169,7 @@ extension EmbeddedGRPCChannel: @unchecked Sendable {}
 
 private final class EmbeddedGRPCChannel: GRPCChannel {
   let embeddedChannel: EmbeddedChannel
-  let multiplexer: EventLoopFuture<HTTP2StreamMultiplexer>
+  let h2Handler: EventLoopFuture<NIOHTTP2Handler>
```

Comment:
Let's change this so we store the `NIOHTTP2Handler.StreamMultiplexer`, we can grab it when we configure the channel below.